src/commands/utility/ping.js

```diff
@@ -1,10 +1,11 @@
 const { SlashCommandBuilder } = require('discord.js');
 const { translateLanguage } = require('../../languages');
+const { botLanguage } = require('../../config');
 
 module.exports = {
   data: new SlashCommandBuilder()
     .setName('ping')
-    .setDescription(translateLanguage('es', 'pingCommand.description')),
+    .setDescription(translateLanguage(botLanguage, 'pingCommand.description')),
```

Comment:
you can avoid importing botLanguage each time you need to use it, if you import it inside the function translateLanguage